### PR TITLE
Site: improve alt text for previewing a site

### LINKED ISSUE #6433: Site Preview link in My Sites Menu is potentially confusing
New users may find it unclear what the sitecard link in the My Sites menu does.

Currently it opens a preview modal with a tooltip that says, ""View [sitetitle]."

**Suggested improvements:**
1. Add a descriptive label like, "Preview your site."
2. Make the tooltip more direct by changing it to, "Preview your site."

### PR DESCRIPTION
**Pull request by lancewillett**
Fixes #6433

Moved from #9086 props @jchimienti89

To test:

1. Open the Live Branch (or test locally)
2. Go to Sites list
3. Hover mouse over sites in the list — should see `Open this site` as alt text
4. Click into a site
5. Hover mouse over the site title — should see `Preview this site` as alt text

### CODE REVIEW

**Comment by lancewillett** [comment]
Visual record:

**Before** screenshots;

screen shot 2016-11-02 at 10 46 38
screen shot 2016-11-02 at 10 46 42

**After** screenshots:

screen shot 2016-11-02 at 10 45 03
screen shot 2016-11-02 at 10 45 16

**Comment by lancewillett** [comment]
Ping @ianstewart for a review.

**Comment by jchimienti89** [comment]
Thanks Lance,

I like the clear testing directions and what you are expecting to happen, I'll have to do that in the future.

**Comment by mtias** [comment]
I'm not sure "preview" is the best word here. Also, Jetpack sites currently open in a new tab which is more clearly a "view site" action.

**Comment by lancewillett** [comment]
@mtias I can add a case for Jetpack sites — but "preview" is indeed more accurate than "view" since it's an overlay and not a new tab open, for WordPress.com sites.

File: client/blocks/site/index.jsx [hunk 1]
```diff
@@ -203,8 +203,8 @@ export default React.createClass( {
 							data-tip-target={ this.props.tipTarget }
 							target={ this.props.externalLink && ! this.state.showMoreActions && '_blank' }
 							title={ this.props.homeLink
-								? this.translate( 'View "%(title)s"', { args: { title: site.title } } )
-								: site.title
+								? this.translate( 'Preview this site' )
+								: this.translate( 'Open this site' )
```
Comment (mtias) [inline client/blocks/site/index.jsx]: I think this case should be "select" instead of "open".

Comment (lancewillett) [inline client/blocks/site/index.jsx]: Good call, that's more accurate.

**Comment by lancewillett** [comment]
Trying out "View" to cover both Jetpack and WP.com site view and "Select" to choose a site in the sites list.

**Comment by lancewillett** [comment]
@ianstewart 2nd opinion on wording of "View" versus "Preview"?

**Comment by ianstewart** [comment]
View works for me.

**Comment by mtias** [comment]
Thanks for the tweaks!

**Pull request merged**